DayrSuryan/DSEthiop15.xml

```diff
@@ -161,35 +161,35 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                          <locus from="3ra" to="3rb"/>
                          <locus from="3va" to="7va"/>
                          <locus from="8vb" to="9ra"/>
-                        <seg type="script">20th century.</seg>
+                        <seg type="script">20th-century cursive script.</seg>
+                          <persName role="scribe" ref="PRS12977WaldaTaklaHaymanot">Walda Takla Hāymānot</persName>
                         <seg type="ink">Black, red.</seg>
                         <seg type="rubrication"><foreign xml:lang="gez">ማርያም፡</foreign>, first word of stanzas,
                            parts of punctuation signs and numerals, <foreign xml:lang="gez">ስብሐት</foreign> and
                            personal names in the supplication at the
                         end.</seg>
                         <date notBefore="1920" notAfter="1953">20th century</date>
-                        <desc>The script is tall, right sloping,
+                        <desc>Careless, hasty hand. The script is tall, right sloping,
                         but very irregular, with changing slant;
-                          the letters are poorly shaped.
-                           Written by a careless, hasty hand.</desc>
+                          the letters are poorly shaped.</desc>
                      </handNote>
 
                      <handNote xml:id="h2" script="Ethiopic">
                          <locus target="#3rb"/>
                          <locus from="7va" to="8vb"/>
-                        <seg type="script">20th century.</seg>
+                        <seg type="script">20th-century cursive script.</seg>
                         <seg type="ink">Black, red.</seg>
                         <seg type="rubrication"><foreign xml:lang="gez">ማርያም፡</foreign>, first word of stanzas, parts of
                           punctuation signs and numerals, <foreign xml:lang="gez">ስብሐት</foreign> and
                            personal names in the supplication at the
                         end.</seg>
                         <date notBefore="1920" notAfter="1953">20th century</date>
-                        <desc>The script is tall, upright to slightly right sloping,
-                          with more rounded shapes than <ref target="#h1">Hand 1</ref>,
-                          more regular than the script of <ref target="#h1">Hand 1</ref>, executed by
-                          a more skilled scribe. </desc>
 
-                     </handNote>
+                        <desc>More careful and regular than <ref target="#h1">Hand 1</ref>.
+                          The script is tall, upright to slightly right sloping,
+                          with more rounded shapes than <ref target="#h1">Hand 1</ref>,
+                          more regular than the script of <ref target="#h1">Hand 1</ref>.</desc>
```

Comment:
Dot after "than <ref target="#h1">Hand 1</ref>", the rest ("more regular...") can be dropped.

Reply:
thank you!